modules/search/instant-search/components/search-results.jsx

```diff
@@ -12,6 +12,8 @@ import { h, Component } from 'preact';
 import SearchResult from './search-result';
 import { hasFilter } from '../lib/query-string';
 import ScrollButton from './scroll-button';
+import SearchForm from './search-form';
+import SearchResultsEmpty from './search-results-empty';
 
 class SearchResults extends Component {
 	getSearchTitle() {
```

Comment:
Would we be able to refactor the `SearchResultsEmpty` component into this function?

Reply:
I had thought the same. When I originally broke it out it had some extra rendering logic, but it's so simple now it can probably just be part of `SearchResults`.